lib/zendesk_apps_tools/deploy.rb

```diff
@@ -17,6 +17,15 @@ def deploy_app(connection_method, url, body)
       say_error_and_exit e.message
     end
 
+    def check_app_installed(url)
```

Comment:
Considering this method is simply making a GET request with the given url, it doesn't seem to be named appropriately. How about having the API path defined here and only passing the `app_id` as an argument?

Reply:
agree, this method will make more sense once it loops through the installations trying to find the `app_id` which should be passed in.